src/content/articles/en/how-to-slack.mdx

```diff
@@ -0,0 +1,187 @@
+
+It might even be interesting to have separate channels for teams of one. In a company no-one will work entirely on their own so people taking over different roles can join those channels as well.
+
+<Note>
+  Make sure that you establish a naming convention for your channels. This will
+  make sure that anybody can channels themselves instead of having to ask around
+  all the time.
+</Note>
+
+### Topic based channels
+
+In the team channels, you will most likely discuss day to day topics but for some projects just having a chat in threads won't be enough anymore. At that point, you might want to move that conversation its own channel: `#project-rebrand` or `#project-conference`
+
+Make sure that after the topic or project is done, the long-term value is moved to the knowledge base and the channel is archived accordingly.
+
+### Announcement channels
+
+You should have at least one channel that is set to read-only for most of the people and is going to be used as a notification channel. This is going to replace the email that will be sent to everyone and everyone replying to everyone. Normally, the `#general` is a good place for this since everybody new to a workspace will be added to this channel anyway. But if you need other specific announcement channels, do add them.
+
+Example `#announcement-engineering`
+
+### The Daily channel
+
+In many distributed teams there is also a daily channel which is mostly used to check-in and check-out. Whenever someone starts to work and signs off for the day, they will post in there. That way everybody will be aware of who is available and who not. This channel is also a good place to say if you are off for running an errand.
+
+Example `#daily-engineering`
+
+### Knowledge sharing channels
+
+If you are working from an office. you will have those conversations in which you learn about that new blog post or a video that will help you with the thing that you are currently working on. Or you just want to share some new insights from the last experiment that you wrote up in the knowledge base.
+
+Example: `#tech-blogs`
+
+### Random
+
+Even in a remote company working should be fun. In the office, you will have those silly conversation and all the fun. You should have them in a distributed team as well. Have channels (yes more than one) for all these random things.
+
+Examples: `#random` `#catlovers` `#rundayfunday` `#quotes`
+
+But you can and should also have channels to share some love. What about sharing customer success stories or appreciate your colleagues work?
+
+Example: `#karma` `#customer-love`
+
+## Productivity tricks and tips
+
+You see there are a some things to consider when you set up Slack. But all of them are worth it. The earlier you work on to get the Slack workspace organized the easier it will be and the more productive your company will be.
```

Comment:
```suggestion
You see there are some things to consider when you set up Slack. But all of them are worth it. The earlier you work on to get the Slack workspace organized the easier it will be and the more productive your company will be.
```